src/v2/Apps/Auction/Routes/ConfirmBid/index.tsx

```diff
@@ -110,9 +117,30 @@ export const ConfirmBidRoute: React.FC<ConfirmBidProps> = props => {
     logger.error(error)
     trackConfirmBidFailed([`JavaScript error: ${error.message}`])
     actions.setSubmitting(false)
-    actions.setStatus(
-      "Something went wrong while processing your bid. Please make sure your internet connection is active and try again"
-    )
+    actions.setStatus("ERROR")
```

Comment:
If I'm understanding the code correctly, it seems like `onJsError` is really used to handle network errors that originate from calls to MP (and `stripe.createToken` in one case), like a connection timeout error. Is it fair to say that this function is used for all unhandled error now, while `handleMutationError` is used for all errors that aren't network errors. Just want to make sure I'm understanding this correctly,

If this is true, would `handleNetworkError` or related be a more accurate method name?

Reply:
A JavaScript error isn't necessarily always a network error. It could an error raised by a dependency due to a bug in it, in which case we never know what the error type is until it happens.

On another thought. we may not have to handle these generic JS errors by ourselves since [the more generic `ErrorBoundary`](https://github.com/artsy/force/blob/ca519287b927e6c97ee3b7e364e54458b92fb46f/src/v2/Artsy/Router/ErrorBoundary.tsx#L19) should catch them automatically. In fact, the error modal we have been in the integrity test failure was actually the one in `ErrorBoundary`, not the one we added explicitly. I wonder how safe it is to remove it entirely from the auctions routes.

cc @damassi